client/app/components/attendees/attendees.module.js

```diff
@@ -0,0 +1,5 @@
+angular
+  .module('components.attendees', []);
+
+require('./attendeesList/attendeesList.component');
```

Comment:
For now, we should let it like that. We will add the guide lines for file names and code later.